development/build/scripts.js

```diff
@@ -1279,20 +1282,24 @@ function renderJavaScriptLoader({
     (label) => `./${label}.js`,
   );
 
-  const securityScripts = applyLavaMoat
-    ? ['./runtime-lavamoat.js', './lockdown-more.js', './policy-load.js']
-    : [
-        './lockdown-install.js',
-        './lockdown-run.js',
-        './lockdown-more.js',
-        './runtime-cjs.js',
-      ];
+  let securityScripts = [
+    './lockdown-install.js',
+    './lockdown-run.js',
+    './lockdown-more.js',
+    './runtime-cjs.js',
+  ];
+
+  if (applyLavaMoat) {
+    securityScripts = ['./lockdown-more.js', './policy-load.js'];
+    if (overrideLavamoatRuntime === false) {
+      securityScripts.unshift('./runtime-lavamoat.js');
+    }
```

Comment:
I had thought maybe the protection of lavamoat somehow carried over to the offscreen document because its a new dom/window but not the same as a usual window. This didn't seem to be the case, not having this yielded the same results.